insalan/tournament/views.py

```diff
@@ -269,11 +276,18 @@ def post(this, request, *args, **kwargs):
                     ]
                 }
             )
-        mut = data._mutable
-        data._mutable = True
+        
+        if data["password"] != Team.objects.get(pk=data["team"]).get_password() :
+            return Response(
+                { "password": _("Mot de passe invalide.")},
+                status=status.HTTP_400_BAD_REQUEST
+            )
```

Comment:
Storing clear text passwords is a really bad security practice, even if they seem "unimportant" (because people tend to use the same password everywhere, we don't want to be the reason one's password is leaked)

Reply:
Yes, I will make it encrypted